### PR TITLE
Mhv 58859 new breadcrumbs structure

### PR DESCRIPTION
**Pull request by fazilqa**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- This PR adds new tests to verify updated breadcrumbs structure for all pages.

## Related issue(s)

- https://jira.devops.va.gov/browse/MHV-58859

## Testing done

- Relevant tests have been executed in headed mode with a 200x pattern.
- All tests have been successfully conducted in headless mode without any failures.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-draft-folder.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import SecureMessagingSite from './sm_site/SecureMessagingSite';
+import PatientInboxPage from './pages/PatientInboxPage';
+import PatientMessagesSentPage from './pages/PatientMessageSentPage';
+import { AXE_CONTEXT, Data } from './utils/constants';
+import FolderLoadPage from './pages/FolderLoadPage';
+
+describe('secure Messaging Sent Folder checks', () => {
```
Comment (oleksii-morgun) [inline src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-draft-folder.cypress.spec.js:7]: file name states draft folder, but a test description includes Sent Folder.

File: src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-draft-folder.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  });
+  it('verify breadcrumbs', () => {
+    cy.injectAxe();
+    cy.axeCheck(AXE_CONTEXT);
+
+    FolderLoadPage.verifyBreadCrumbsLength(4);
+    FolderLoadPage.verifyBreadCrumbText(0, 'VA.gov home');
+    FolderLoadPage.verifyBreadCrumbText(1, 'My HealtheVet');
+    FolderLoadPage.verifyBreadCrumbText(2, 'Messages');
+    FolderLoadPage.verifyBreadCrumbText(3, 'Drafts');
```
Comment (oleksii-morgun) [inline src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-draft-folder.cypress.spec.js:45]: if the test is on Sent folder, is this line above valid?

File: src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-folders.cypress.spec.js [hunk 3]
```diff
@@ -13,38 +13,36 @@ describe(manifest.appName, () => {
...
-        'aria-required-children': {
-          enabled: false,
-        },
-      },
-    });
+    cy.axeCheck(AXE_CONTEXT);
     FolderLoadPage.verifyFolderHeaderText('Trash');
-    FolderLoadPage.verifyBackToMessagesButton();
+    FolderLoadPage.verifyBreadCrumbsLength(4);
+    FolderLoadPage.backToFolder('Messages');
```
Comment (oleksii-morgun) [inline src/applications/mhv-secure-messaging/tests/e2e/secure-messaging-folders.cypress.spec.js:46]: should we consider including `FolderLoadPage.backToFolder('Messages');` in `afterEach` ?

**Review by alex-morgun-1 — CHANGES_REQUESTED** [review]
I did leave some comments and couple suggestions. Are we considering scenarios for Compose, Reply and Thread pages? there is a different behavior when navigating to a those pages via a direct url which will be different if you use UI to navigate

**Pull request merged**